tendrl/node_agent/node_sync/__init__.py

```diff
@@ -163,7 +163,7 @@ def _run(self):
                             # it will delete subnet dir when it is empty
                             # if one entry present then deletion never happen
                             NS.etcd_orm.client.delete(network.key, dir=True)
```

Comment:
@r0h4n all the network interface are stored under subnet, if all interface are down in particular subnet then subnet path contains empty  and and empty subnet path always present etcd. so it will delete subnet when subnet path completely empty